src/store/store.ts

```diff
@@ -1,4 +1,5 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-cycle
```

Comment:
If you have to do this, probably something isn't right

Reply:
We just need common types file in the root and we'll not have such errors related to cycling